server/model/monitor.js

```diff
@@ -231,7 +231,11 @@ class Monitor extends BeanModel {
                             bean.msg += ", keyword is found";
                             bean.status = UP;
                         } else {
-                            throw new Error(bean.msg + ", but keyword is not found");
+                            data = data.replace(/<[^>]*>?|[\n\r]|\s+/gm, " ");
+                            if ( data.length > 50 ) {
+                                data = data.substring(0, 47) + "...";
+                            }
+                            throw new Error(bean.msg + ", but keyword is not in [ " + data + " ]");
```

Comment:
Stay in this same "spaces formatting"
![image](https://user-images.githubusercontent.com/905878/151350779-28f9cc3a-0361-44c3-abf4-9b3c49ca5f25.png)


```suggestion
                            throw new Error(bean.msg + ", but keyword is not in [" + data + "]");
```